source/nodejs/adaptivecards-designer/src/dialog.ts

```diff
@@ -99,10 +103,13 @@ export abstract class Dialog {
             titleElement.style.flex = "1 1 auto";
 
             let xButton = document.createElement("button");
-            xButton.className = "acd-icon acd-dialog-titleBar-button acd-icon-remove";
+            xButton.className =
```

Comment:
Can we put this statement on a single line?
EDIT: I'm going to assume this is the result of automatic formatting?

Reply:
@dclaux Yes. We have used [Prettier for VS code](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) as default. Let me know if there are any custom formatting settings used for Adaptive Cards. Will use the same hereafter.
Addressed the review comments by changing to single line.